README.md

```diff
@@ -7,9 +7,12 @@ Hardware and Software Requirements
 ----------------------------------
 Steam for Linux requires the following:
 
-- 1 GHz Pentium 4 or AMD Opteron with 512 megabytes of RAM and 5 gigabytes of hard drive space, or better
+- 1 GHz Pentium 4 or AMD Opteron with x86-64 (AMD64) instruction set
```

Comment:
According to #4196, x86-64 CPUs without `cmpxchg16b` support (`cx16` in `/proc/cpuinfo`) are also unsupported, but I'm not sure how to describe that requirement in user-facing terms. It's missing from very early AMD x86-64 CPUs, but seems to be implemented in all Intel x86-64 CPUs? The affected CPUs are the same ones that can't run 64-bit Windows 8.1.